app/src/pages/[resourceType]/[id].tsx

```diff
@@ -80,6 +98,50 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     }
   );
 
+  //Sorts dependencies alphabetically based on their display property. If the the
+  //dependency is a library/measure, however, it will always take precedence in sorting to ensure all
+  //dependencies with links are always shown first
+  const sortDependencies = useCallback(() => {
+    if (dataRequirements?.relatedArtifact) {
+      dataRequirements.relatedArtifact.sort((firstElement, secondElement) => {
+        const displayFirst = firstElement.display?.toUpperCase();
+        const displaySecond = secondElement.display?.toUpperCase();
+        if (displayFirst && displaySecond) {
+          if (
+            firstElement.resource?.includes('Library') ||
+            firstElement.resource?.includes('Measure') ||
+            secondElement.resource?.includes('Library') ||
+            secondElement.resource?.includes('Measure')
+          ) {
+            return 1;
```

Comment:
Good point! I think originally, the reason as to why I did this was because I was thinking it might be best to place all of the resources that have links at the very top so the user can easily select them. However, I didn't sort the resources with links alphabetically so I edited my current code to reflect that. I was curious though, do you think it would be better to sort everything solely alphabetically or to place the artifacts with links first and then sort everything else after? I have screenshots of both attached: 

Dependencies with links placed first:
![Screenshot 2023-07-12 at 1 20 05 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/fcc7cb3c-e692-44a5-8190-68e41b2cd4af)

Dependencies sorted regularly: 
![Screenshot 2023-07-12 at 1 21 25 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/eaaafeba-2089-4678-8001-db972261aaee)

Reply:
I think that artifacts with links first is a good idea! Option A looks good to me :-)